backend/twitterbot/uploader_twitter_account.py

```diff
@@ -0,0 +1,34 @@
+"""
+Function to get twitter account of the uploader of a video.
+Used for the Tournesol twitter bot.
+"""
+
+import re
+import requests
+from tournesol.utils.api_youtube import get_video_metadata
+
+
+def get_twitter_account_from_channel_id(channel_id):
+    """Get Twitter account from uploader id"""
+
+    uploader_url = f"https://www.youtube.com/channel/{channel_id}"
+
+    resp = requests.get(uploader_url, headers={"user-agent": "curl/7.68.0"})
+
+    twitter_names = re.findall(r"twitter.com\%2F(.*?)\"", resp.text)
+    twitter_names = [name for name in twitter_names if name != ""]
+
+    if len(set(name.lower() for name in twitter_names)) == 1:
```

Comment:
set comprehension is prefered as generator expresion casted as set
```suggestion
    if len({name.lower() for name in twitter_names}) == 1:
```

Reply:
Thanks